src/app/view/components/dot-navigation/components/dot-nav-item/dot-nav-item.component.scss

```diff
@@ -69,7 +69,7 @@
     display: flex;
     justify-content: space-between;
     min-height: $navigation-item-height;
-    padding: 14px $basic-padding * 3 13px 23px; // top and bottom padding to get min 48px height
+    padding: 14px 22px 13px 25px; // top and bottom padding to get min 48px height
```

Comment:
We can't hardcode this to pixels because here you can have FA icons but also MD so we need to find a way to make both to be centered.